internal/rqlite/rqlite.go

```diff
@@ -288,8 +288,9 @@ func (r *RQLite) Dequeue(serverID string, qnames ...string) (msg *base.TaskMessa
 	}
 
 	for _, qname := range qnames {
-
+		t := time.Now()
 		q, err := r.getQueue(qname)
+		r.logger.Debug(fmt.Sprintf("getQueue [%s], qname=%s, err=%v", time.Since(t).String(), qname, err))
```

Comment:
Do you want to wait satisfying tests and remove these debug logs before merging ?

Reply:
Yeah, we can wait; I wanted to remove the logs after we don't need them anymore